hdf5-sys/build.rs

```diff
@@ -221,6 +231,7 @@ fn get_conf_header<P: AsRef<Path>>(inc_dir: P) -> PathBuf {
     }
 }
 
+#[must_use]
```

Comment:
`#[must_use]` is very questionable in our case, and clippy would recommend every single thing to be marked with it - I'd say add a lint ignore for it and remove must-uses?

Reply:
Blacklisted and removed `#[must_use]`